model/src/main/proto/interaction_object.proto

```diff
@@ -53,5 +54,30 @@ message Fraction {
 
 // Structure for a ListOfString object.
 message ListOfSetsOfHtmlStrings {
-  repeated StringList set_of_html_strings = 1;
+  StringList set_of_html_strings = 1;
+}
+
+// Structure for a ImageWithRegions object.
+message ImageWithRegions {
+  string image_path = 1;
```

Comment:
Please add documentation. What does ``path`` here mean? Is it the absolute file to the image being stored, the file name, a URL, a relative path, something else? Also, why do we need to store the path in the interaction object? Isn't this a customization of the interaction and not something the user or creator selects for a specific answer?